contracts/ERC721Holder.sol

```diff
@@ -38,3 +34,49 @@ contract ERC721Holder is TokenHolder {
         }
     }
 }
+
+interface IERC721Minter is IERC721 {
+    function mint(address to, uint256 id) external;
+}
+
+contract ERC721MintableHolder is ERC721Holder {
+    using Bytes for bytes;
+
+    constructor(address erdstall) ERC721Holder(erdstall) {}
+
+    function transfer(address _token, address recipient, bytes calldata value)
+    override external onlyErdstall
+    {
+        IERC721Minter token = IERC721Minter(_token);
+        uint256[] memory ids = value.asUint256sInplace();
+        for (uint i=0; i < ids.length; i++) {
+            transferOrMint(token, recipient, ids[i]);
+        }
+    }
+
+    function transferOrMint(IERC721Minter token, address to, uint256 id) internal {
+        try token.ownerOf(id) returns (address owner) {
+            if (owner == address(this)) {
+                token.safeTransferFrom(address(this), to, id);
+            }
+            // else token exists, but owned by someone else, so we skip
+            // minting it. In a production setting, there has to be some
+            // mechanism in place to guarantee that off-chain minted tokens
+            // are not double-minted.
```

Comment:
Is there an issue already existing which mentions this?
If not I would create it either way and mention it here again for reference, just to be sure we do not lose track :railway_track:

Reply:
Feel free to resolve this, since it is an example anyway :D

Reply:
It's not something that _we_ can solve alone in this repo. This is an observation that has to go into the documentation of how to create off-chain mintable NFT contracts...

It's a good idea to `revert` here once we've done #8. Because we haven't done this yet is the reason why I didn't revert here yet.

Reply:
> Feel free to resolve this, since it is an example anyway :D

Well, the `PerunArt` contract is an example, _this_ contract is potentially also used in production. So it's good that you keep an 👁️ out!

Reply:
Jetzt wird ein Schuh draus 💡

Thanks for clarifying.